### PR TITLE
[authorization] update doc about roleRef

### PR DESCRIPTION
**Pull request by CaoShuFeng**
This update is made according to this commit in code:
https://github.com/kubernetes/kubernetes/commit/8c788233e778f3b0ebef560762c1433c12ea1d43

It has already be released  with k8s v1.5, so it's reasonable to
update the doc now.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1888)

### CODE REVIEW

**Comment by deads2k** [comment]
@devin-donnelly this actually needs to be against the 1.5 branch too, right?

File: docs/admin/authorization.md [hunk 1]
```diff
@@ -336,14 +335,14 @@ namespaces. The following `ClusterRoleBinding` allows any user in the group
...
-  name: read-secrets
+  name: read-secrets-global
 subjects:
   - kind: Group # May be "User", "Group" or "ServiceAccount"
     name: manager
 roleRef:
   kind: ClusterRole
-  name: secret-reader
-  apiVersion: rbac.authorization.k8s.io/v1alpha1
+  name: secret-reader-global
```
Comment (deads2k) [inline docs/admin/authorization.md]: The role didn't change names, so this doesn't look right.

Comment (CaoShuFeng) [inline docs/admin/authorization.md]: done

**Comment by deads2k** [comment]
lgtm, needs to squash down.

**Comment by devin-donnelly** [comment]
@deads2k no, 1.5 has launched, so that branch has merged and is effectively defunct. Master is fine.

**Pull request merged**